tower_cli/api.py

```diff
@@ -223,12 +236,12 @@ def request(self, method, url, *args, **kwargs):
         # Sanity check: Did we fail to authenticate properly?
         # If so, fail out now; this is always a failure.
         if r.status_code == 401:
-            raise exc.AuthError('Invalid Tower authentication credentials.')
+            raise exc.AuthError('Invalid Tower authentication credentials (HTTP 401).')
```

Comment:
This is beyond the scope of this issue, and PR, but I think it may take debugging a bit clearer for folks (at least it did for me when dealing w/ read vs. write oauth2 scopes).